src/shimoku_api_python/resources/reports/charts/EChart_definitions/scatter.py

```diff
@@ -39,5 +40,54 @@ async def scatter_chart(
         echart_options=common_options, series_options=series_options, option_modifications=option_modifications,
         axes=[], values=point_fields, data_mapping_to_tuples=await self._choose_data(order, data),
         order=order, x_axis_names=x_axis_name, y_axis_names=y_axis_name, title=title,
-        rows_size=rows_size, cols_size=cols_size, padding=padding, show_values=show_values
+        rows_size=rows_size, cols_size=cols_size, padding=padding, show_values=show_values, variant=variant
+    )
+
+
+@async_auto_call_manager()
+@logging_before_and_after(logger.info)
+async def scatter_with_effect_chart(
+        self: 'PlotApi', order: int, x: str, y: str,
+        data: Optional[Union[List[str], DataFrame, List[Dict]]],
+        x_axis_name: Optional[str] = None, y_axis_name: Optional[str] = None,
+        title: Optional[str] = None, rows_size: Optional[int] = None, cols_size: Optional[int] = None,
+        padding: Optional[List[int]] = None, effect_points: Optional[List] = None,
+        option_modifications: Optional[Dict] = None,
+):
+    """ Create a scatter chart """
+    data_mappings_to_tuples = await self._choose_data(order, data)
+    if isinstance(data, str):
+        data = self._shared_data[data]
+
+    df: DataFrame = validate_data_is_pandarable(data)
+    df = df[[x, y]].dropna()
+    matrix = df.values
+
+    common_options = get_common_echart_options()
+    x_axis = common_options['xAxis'][0]
+    del x_axis['data']
+    x_axis['type'] = 'value'
+    x_axis['scale'] = True
+    common_options['yAxis'][0]['scale'] = True
+    del common_options['toolbox']['feature']['magicType']
+    common_options['series'] = [
+        {
+            'data': [(point if isinstance(point, list) else list(matrix[point]))
+                     for point in effect_points],
+            'type': 'effectScatter',
+            'symbolSize': 20,
+        },
+        {
+            'type': 'scatter',
+            'itemStyle': {
+                'color': 'var(--chart-C2)',
```

Comment:
Maybe is a good idea to move this colors to a exportable constants

Reply:
I could make an enum for the colors that are defined there, in the utils module